validation/Makefile

```diff
@@ -18,7 +18,12 @@ $(CLIENTS_DIR):
 
 # build all cert chains
 $(CERTS_DIR): $(CLIENTS_DIR)
-	@make --no-print-directory \
+	@python -m http.server --bind "localhost" \
+	                       --directory "$(CURDIR)/$(CERTS_DIR)/build/static" \
```

Comment:
It would be nice if the `build/static` directory wasn't just hardcoded here and in `validate.sh`. However, there are similar issues caused by me (and waiting for me to fix them), so you don't need to mess with the variables/arguments unless you really want to.

Reply:
Does not need to be addressed now, noted as #122.